### DISCUSSION TITLE
DKI info

### DISCUSSION THREAD

**Discussion by Yvainnn** [dipy/dipy#3353] [Reconstruction] [answered]
Hi everyone, 
I'm working on dipy for the first time, specifically on MSDKI-DKI estimation. I tried to run it on my data a couple of times and now I'm optimizing my pipeline ecc... I have a question about model fitting, I tried to read about it on documents/tutorial but I can't understand how I can fit it to my sequence (b1000/b2000/b2800 - tot: 192 direc)
![image](https://github.com/user-attachments/assets/31916c4d-c7b7-4657-af1b-6a98aede4714)
What are the differences between :

_MK = dki_fit.mk()_

instead of 

MK = dki_fit.mk(0,3) 

or 

_MK = dki_fit.mk(0,192) 
MK = dki_fit.mk(1,192) 
MK = dki_fit.mk(2,192)_ 
ecc....

Can someone explain to me how I can use all the available directions / data to estimate the mean kurtosis?
Thank you so much !

**Comment by arokem** [comment, marked as answer]
Hello! You might want to take a look at this example: https://docs.dipy.org/stable/examples_built/reconstruction/reconst_dki.html#sphx-glr-examples-built-reconstruction-reconst-dki-py

**Reply by skoudoro** [reply]
Also at the documentation of `mk` (mean kurtosis) function:  https://docs.dipy.org/stable/reference/dipy.reconst.html#dipy.reconst.dki.DiffusionKurtosisFit.mk

- `MK = dki_fit.mk()`  means `dki_fit.mk(min_kurtosis=-0.42857142857142855, max_kurtosis=10)`
- `MK = dki_fit.mk(0, 3)`  means `dki_fit.mk(min_kurtosis=0, max_kurtosis=3)`

Let us know if you have more question @Yvainnn

**Reply by Yvainnn** [reply]
Thank you much !!
